plugins/woocommerce/includes/wc-core-functions.php

```diff
@@ -1592,7 +1594,7 @@ function wc_help_tip( $tip, $allow_html = false ) {
 	 *
 	 * @return string
 	 */
-	return apply_filters( 'wc_help_tip', '<span class="woocommerce-help-tip" tabindex="0" aria-label="' . $sanitized_tip . '" data-tip="' . $sanitized_tip . '"></span>', $sanitized_tip, $tip, $allow_html );
+	return apply_filters( 'wc_help_tip', '<span class="woocommerce-help-tip" tabindex="0" aria-label="' . $aria_label . '" data-tip="' . $sanitized_tip . '"></span>', $sanitized_tip, $tip, $allow_html );
```

Comment:
```suggestion
	return apply_filters( 'wc_help_tip', '<span class="woocommerce-help-tip" tabindex="0" aria-label="' . esc_attr( $aria_label ) . '" data-tip="' . $sanitized_tip . '"></span>', $sanitized_tip, $tip, $allow_html );
```

It's generally best practice to escape as late as possible, so this seems like an opportunity to improve the existing markup implementation here and move the `esc_attr` into this markup string here (and remove it from line 1582 above), at least for `$aria_label`. (I realize that `wc_sanitize_tooltip` says it shouldn't be used in conjunction with `esc_attr`.)